components/infobox/wikis/deadlock/infobox_character_custom.lua

```diff
@@ -0,0 +1,98 @@
+---
+-- @Liquipedia
+-- wiki=deadlock
+-- page=Module:Infobox/Character/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+
+local Lua = require('Module:Lua')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector')
+local Character = Lua.import('Module:Infobox/Character')
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Title = Widgets.Title
+
+---@class DeadlockHeroInfobox: CharacterInfobox
+local CustomHero = Class.new(Character)
+local CustomInjector = Class.new(Injector)
+
+---@param frame Frame
+---@return Html
+function CustomHero.run(frame)
+	local character = CustomHero(frame)
+	character:setWidgetInjector(CustomInjector(character))
+	character.args.informationType = 'Hero'
+	if character.args.damagebullet and character.args.bps then
```

Comment:
maybe numeric checks?
like theoretically bad input (non numeric) could be a thing

Reply:
I'm fine with errors on shit input :P